crypt/secret_sharer.go

```diff
@@ -7,8 +7,39 @@ import (
 	"crypto/sha256"
 	"errors"
 	"math/big"
+	"sort"
 )
 
+/////// Share sorting ////////
+type By func(p1, p2 *SecretShare) bool
```

Comment:
1. why this needs to be a func (as opposed to just a struct?)
2. no need for a capital B - this is for internal usage within the pacakge